netlify/edge-functions/card.tsx

```diff
@@ -202,6 +261,42 @@ export default async function handler(request: NextRequest) {
   );
 }
 
+function getRepoList(repos: string) {
+  return repos
+    .split(",")
+    .filter((rpo) => !!rpo)
+    .map((repo) => {
+      const [, repoName] = repo.split("/");
+
+      return {
+        repoName,
+      };
+    });
+}
+
+const getAvatarByUsername = (username: string | null, size = 460) =>
+  `https://www.github.com/${username ?? "github"}.png?size=${size}`;
+
+async function fetchContributorPRs(username: string) {
+  const query = new URLSearchParams();
+
+  query.set("topic", "*");
+  query.set("limit", "100");
+  query.set("range", "30");
+
+  const baseEndpoint = `users/${username}/prs`;
+  const endpointString = `${baseEndpoint}?${query.toString()}`;
+  const url = `https://beta.api.opensauced.pizza/v1/${endpointString}`;
```

Comment:
How do we switch to the prod API here?

```suggestion
  const url = `https://beta.api.opensauced.pizza/v1/${endpointString}`;
```